llvm/lib/Target/RISCV/RISCVInstrInfoXCV.td

```diff
@@ -89,6 +91,7 @@ class CVInstMacMulN<bits<2> funct2, bits<3> funct3, dag outs, dag ins,
   let DecoderNamespace = "XCV";
 }
 
+let hasSideEffects = 0, mayLoad = 0, mayStore = 0 in {
```

Comment:
These can, I think, go inside `CVInstMacMulN`

Reply:
I didn't do that because I thoguth `CVInstMacMulN` was more like a format class like RISCVInstrFormats.td rather than a class in RISCVInstrInfo.td. So the flags would go on its derived classes if I keep the usual separation.